includes/class-wc-gateway-stripe.php

```diff
@@ -294,6 +294,22 @@ public function payment_fields() {
 			$pay_button_text = '';
 		}
 
+		/**
+		 * Start a payment intent.
+		 *
+		 * @see https://stripe.com/docs/payments/payment-intents/quickstart
+		 */
+		$request = array(
+			'amount'               => round( $total * 100 ),
```

Comment:
Use [`WC_Stripe_Helper::get_stripe_amount`](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/453f739d2df7316f1a60aeb37e8036a337de903a/includes/class-wc-stripe-helper.php#L186) instead, or else we'll have to deal with some angry Japanese users.